config/deploy.rb

```diff
@@ -0,0 +1,41 @@
+# config valid only for current version of Capistrano
+lock '3.4.0'
+
+set :application, 'sufia'
```

Comment:
In general can we use 'chf-sufia' or something that otherwise makes it clear semantically that we are installing an app based on the sufia engine (as opposed to sufia itself, which can't be run as an app because it is a gem / engine)?